src/components/products/productItem.css

```diff
@@ -0,0 +1,18 @@
+.card-content img{
+    height: 100px;
+    width: 100px; 
+  
+}
+
+.card-content h3{
+   font-size: 13px;
+}
+.card-content p{
+    font-size: 10px;
+}
+.card-content .card-button{
+    
+   display: flex;
```

Comment:
Avoid using flex for easy tasks, like centering a button/image. you could just give the button an automatic horizontal margin (margin: 0 auto)